src/sentry_utils.c

```diff
@@ -339,3 +341,26 @@ sentry__dsn_get_attachment_url(
     sentry__stringbuilder_append(&sb, "/attachments/");
     return sentry__stringbuilder_into_string(&sb);
 }
+
+char *
+sentry__msec_time_to_iso8601(uint64_t time)
+{
+    char buf[255];
+    time_t secs = time / 1000;
+    struct tm *tm;
+#ifdef SENTRY_PLATFORM_WINDOWS
+    tm = gmtime(&secs);
+#else
+    struct tm tm_buf;
+    tm = gmtime_r(&secs, &tm_buf);
+#endif
+    size_t end = strftime(buf, sizeof buf, "%FT%T", tm);
```

Comment:
@Amphaal So for some reason the mingw build returns `0` here, which means that the `snprintf` below overwrites the whole buffer, which in turn means that our tests are failing because of wrong time formatting here:
https://dev.azure.com/sentry-builds/sentry-native/_build/results?buildId=1789&view=logs&j=3e1f1b51-d184-5f5a-906a-d9e9ce13cd6b&t=372e633b-daac-5e1b-39f1-1bfa2728bd78
Do you have any quick idea about that?
I had a talk earlier with @jan-auer and given that none of us has the capacity to actually maintain this, and the problem that it just hangs and times out half of the builds on CI, and the builds generally being slower than all the other platforms.
We might just remove it from CI again, and demote it to *kind of works, but we don’t actively maintain it* status.

Reply:
Giving the context, it's fully understandable. The only way to reduce build time would be to use a precompiled docker image paired with the adequate CI (like Jenkins or CircleCI, I do not believe Azure Pipelines can do that rn). And that LLVM linker bug is really annoying... I'm 100% with you on this one.

Reply:
@Amphaal thanks for your contributions to this! In case you come around to find a good working solution, let us know and we can reenable builds again.